EIPS/eip-draft_getBlockByHash_add_basefee.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: <to be assigned>
+title: Add basefee in eth_getBlockByHash response
+author: Abdelhamid Bakhta (@abdelhamidbakhta)
+discussions-to:
```

Comment:
Once you update the number, I recommend creating a new discussion topic over at https://ethereum-magicians.org/ and pasting the link here.

Reply:
Done